font/language.go

```diff
@@ -0,0 +1,6 @@
+package font
+
+import "github.com/benoitkugler/textlayout/language"
+
+type Language = language.Language
+type Script = language.Script
```

Comment:
My only concern is that these are not inherently a font characteristic, but rather a characteristic of the input text. I'd prefer to move these into a separate `language` package in `typesetting` or put them in `shaping` (since that's where they're consumed from). As it stands, I think accessing them as `font.{Language,Script}` is misleading.

I totally typed this out before, but I think maybe I didn't finalize the comment and then I submitted my review and the draft vanished. Sorry for the confusing review with no context!

Reply:
I see this now, thanks.
Same has happened to me before :)